### DISCUSSION TITLE
[BUG] Achievements UI

### DISCUSSION THREAD

**Discussion by plumkewe** [community/community#43898] [Mobile] [answered]
<summary> 
If you have only one achievement and you open it via new shortcut on the profile and swipe to see other ones it will make the entire sheet (UI) white. 

[Video](https://user-images.githubusercontent.com/113628928/211650404-3c8c875c-5184-4025-9aec-58d6dff05749.MOV)

<summary>ℹ️ Additional Debug Info</summary>

GitHub for iOS, Version 1.95.0 (88456997)
Device: iPhone 13 Pro (iOS 16.2)
</details>

**Comment by HenSquared** [comment, marked as answer]
Hi @plumkewe,
Great find and thanks for filing this issue. 
I'm able to reproduce this on iOS and have a fix in mind! Hope to get this fix in a release soon.
